cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -303,8 +306,27 @@ private Set<ReadOnlyDataSource> separateByModelingAuthority() {
 
         private Optional<String> readModelingAuthority(String name) {
             String modellingAuthority = null;
-            try (InputStream is = dataSource.newInputStream(name)) {
-                XMLStreamReader reader = xmlInputFactory.createXMLStreamReader(is);
+            try (InputStream in = dataSource.newInputStream(name)) {
+                String fileExtension = name.substring(name.lastIndexOf('.') + 1);
+                if (fileExtension.equals(CompressionFormat.ZIP.getExtension())) {
+                    try (SafeZipInputStream zis = new SafeZipInputStream(new ZipInputStream(in), 2048)) {
```

Comment:
Just saw an xml file with a copyright header! It's probably too drastic to put 2048 as a limit here, and to 1024 in the other call, as you might have some comments in the header (like a copyright). Shouldn't we raise it a bit?

Reply:
Which value could be reasonable ? A standard copyright should not take more than 1000 bytes, but there could be additional comments...

Reply:
I agree it should be ok with current parameters, but we should still take some more margin just in case. Plus the values are hardcoded. As the maximum value is only to avoid zip bombing, you could take something like 10kB instead, we're still pretty safe.

Reply:
The overall limit is fine, and in fact that's safer.

Besides right now it's only used for a single entry. If we read several entries in another usecase, the overall  limit would be raise in that use case. And at that time we would also need to improve the SafeZipInputStream to check the number of entries.